src/components/style/form.css

```diff
@@ -0,0 +1,44 @@
+.form-section {
```

Comment:
- [ ] Styling of your app does not conform to the provided template on zeplin. We actually want your design to match exactly the provided template design. Kindly pick the right dimension, colors and other exact values from the [design page](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) on zeplin, and ensure to improve upon the _layout_, _typography_ and _color_ of your design to closely match the provided design.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/185769885-810e8b4c-4c2c-406e-9f30-cb6c3341dcdb.png)


**Original design:**
![image](https://user-images.githubusercontent.com/12745474/185769922-d8cda7d5-8f2f-420b-ac2c-26431dc3b23c.png)